lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -27,6 +27,7 @@ my %URL_SPECIALIZATIONS = (
     'BaiduBaike'          => qr{^https?://baike.baidu.com/}i,
     'Bandcamp'            => qr{^https?://([^/]+.)?bandcamp.com/}i,
     'Bandsintown'         => qr{^https?://(?:www.)?bandsintown.com/}i,
```

Comment:
Maybe we should go back and escape the `.` in these other regexps...

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/670/commits/47c1f91f1b95f0d56f646182e0010a69128aff07